Bluetooth BlueZ Binding fails as Native Libraries not deployed

Working with the current snapshot, after installing the Bluetooth BlueZ Binding and attempting to configure a new Bluetooth BlueZ Adapter, the Thing displays "Status: UNINITIALIZED - HANDLER_INITIALIZING_ERROR BlueZ JNI connection cannot be established.”. The log shows Caused by: java.lang.UnsatisfiedLinkError: tinyb.BluetoothManager.getNativeAPIVersion()Ljava/lang/String;" On checking the code, this error is thrown if there is a problem using the Native Libraries included in the Binding bundle.

## Expected Behavior
That the Thing initialise correctly, without error.

## Current Behavior
When the binding is installed, the included Native Libraries libtinyb.so and libjavatinyb.so are not deployed leading to the error when a Bluetooth BlueZ Adapter is added/configured.

## Steps to Reproduce
1. Install the Bluetooth BlueZ Binding.
2. Add a Bluetooth BlueZ Adapter Thing configured with its MAC.

## Context
The problem prevents a Bluetooth BlueZ Adapter being configured and so prevents any other Things being created that need to use a Bluetooth BlueZ Adapter as a Bridge.

## Environment
* OpenHabian 1.4.1 with OpenHab and OpenHab-Addons upgraded to the current snapshot.

Both libraries are now deploying but things just got worse...

2019-06-08 08:55:12.227 [ERROR] [org.openhab.binding.bluetooth.bluez ] - bundle org.openhab.binding.bluetooth.bluez:2.5.0.201906041543 (229)[org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService(233)] : The activate method has thrown an exception
java.lang.NullPointerException: null

2019-06-08 08:55:12.315 [WARN ] [org.openhab.binding.bluetooth.bluez ] - FrameworkEvent WARNING - org.openhab.binding.bluetooth.bluez
org.osgi.framework.ServiceException: org.apache.felix.scr.impl.manager.SingleComponentManager.getService() returned a null service object

2019-06-08 08:55:12.310 [ERROR] [org.openhab.binding.bluetooth.bluez ] - bundle org.openhab.binding.bluetooth.bluez:2.5.0.201906041543 (229)[org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService(233)] : The activate method has thrown an exception
java.lang.NullPointerException: null

2019-06-08 08:55:12.461 [WARN ] [org.openhab.binding.bluetooth.bluez ] - FrameworkEvent WARNING - org.openhab.binding.bluetooth.bluez
org.osgi.framework.ServiceException: org.apache.felix.scr.impl.manager.SingleComponentManager.getService() returned a null service object

2019-06-08 08:55:12.495 [ERROR] [org.openhab.binding.bluetooth.bluez ] - bundle org.openhab.binding.bluetooth.bluez:2.5.0.201906041543 (229)[org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService(233)] : The activate method has thrown an exception
java.lang.NullPointerException: null

2019-06-08 08:55:12.991 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler@14f929': null
java.lang.NullPointerException: null

2019-06-08 08:55:13.055 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:a07c6635': null
java.lang.NullPointerException: null

2019-06-08 08:55:13.328 [hingStatusInfoChangedEvent] - 'bluetooth:bluez:a07c6635' changed from INITIALIZING to UNINITIALIZED (HANDLER_INITIALIZING_ERROR)

After updating the bundle, and adding the Thing for the Bluez adapter, still getting...

2019-06-08 13:29:16.974 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler@49d3a7': null
java.lang.NullPointerException: null

2019-06-08 13:29:16.999 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:1a299271': null
java.lang.NullPointerException: null

Assuming you do, it looks like this when adding the Adapter...

2019-06-08 15:34:44.637 [DEBUG] [org.openhab.binding.bluetooth.bluez ] - ServiceEvent REGISTERED - {org.openhab.binding.bluetooth.BluetoothAdapter}={service.id=404, service.bundleid=235, service.scope=singleton} - org.openhab.binding.bluetooth.bluez

2019-06-08 15:34:44.710 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler@13bf98d': null

java.lang.NullPointerException: null

	at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:103) ~[?:?]

	at sun.reflect.GeneratedMethodAccessor29.invoke(Unknown Source) ~[?:?]

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]

	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]

	at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [134:org.openhab.core:2.5.0.201906040301]

	at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [134:org.openhab.core:2.5.0.201906040301]

	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]

	at java.lang.Thread.run(Thread.java:748) [?:?]

2019-06-08 15:34:44.735 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:0646642a': null

java.lang.NullPointerException: null

	at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:103) ~[?:?]

	at sun.reflect.GeneratedMethodAccessor29.invoke(Unknown Source) ~[?:?]

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]

	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]

	at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [134:org.openhab.core:2.5.0.201906040301]

	at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [134:org.openhab.core:2.5.0.201906040301]

	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]

	at java.lang.Thread.run(Thread.java:748) [?:?]

That looks really weird. I get:

```
15:16:27.783 [INFO ] [ome.event.ThingStatusInfoChangedEvent] - 'bluetooth:bluez:a721b1dd' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to INITIALIZING
15:16:32.781 [WARN ] [.core.thing.internal.ThingManagerImpl] - Initializing handler for thing 'bluetooth:bluez:a721b1dd' takes more than 5000ms.
15:16:52.796 [ERROR] [rnal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler@769761d4': Cannot access BlueZ layer.
java.lang.IllegalStateException: Cannot access BlueZ layer.
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:105) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [133:org.openhab.core:2.5.0.201906040301]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [133:org.openhab.core:2.5.0.201906040301]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.lang.RuntimeException: Error getting object manager client: Fehler beim Aufruf von StartServiceByName für org.bluez: Zeitüberschreitung wurde erreicht
        at tinyb.BluetoothManager.getNativeAPIVersion(Native Method) ~[?:?]
        at tinyb.BluetoothManager.getBluetoothManager(BluetoothManager.java:304) ~[?:?]
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:93) ~[?:?]
        ... 10 more
15:16:52.816 [ERROR] [.core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:a721b1dd': Cannot access BlueZ layer.
java.lang.IllegalStateException: Cannot access BlueZ layer.
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:105) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [133:org.openhab.core:2.5.0.201906040301]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [133:org.openhab.core:2.5.0.201906040301]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.lang.RuntimeException: Error getting object manager client: Fehler beim Aufruf von StartServiceByName für org.bluez: Zeitüberschreitung wurde erreicht
        at tinyb.BluetoothManager.getNativeAPIVersion(Native Method) ~[?:?]
        at tinyb.BluetoothManager.getBluetoothManager(BluetoothManager.java:304) ~[?:?]
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:93) ~[?:?]
        ... 10 more
15:16:52.833 [INFO ] [ome.event.ThingStatusInfoChangedEvent] - 'bluetooth:bluez:a721b1dd' changed from INITIALIZING to UNINITIALIZED (HANDLER_INITIALIZING_ERROR): Cannot access BlueZ layer.
```

which shows that the initialization code is evil because it blocks for more than 5s but also works. I don't understand why you get a NPE on calling `initialize()`. The only difference I see is that you are on arm while my VM is x86. Not sure how I should debug that.

Can you show `bundle:list -s` once more and try what happens if you restart OH?

Restarted and I get...

2019-06-08 16:55:11.874 [DEBUG] [org.openhab.binding.bluetooth.bluez ] - ServiceEvent REGISTERED - {org.openhab.binding.bluetooth.BluetoothAdapter}={service.id=395, service.bundleid=254, service.scope=singleton} - org.openhab.binding.bluetooth.bluez

2019-06-08 16:55:11.926 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler@151da5a': null

java.lang.NullPointerException: null

	at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:103) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]

	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]

	at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [134:org.openhab.core:2.5.0.201906040301]

	at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [134:org.openhab.core:2.5.0.201906040301]

	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]

	at java.lang.Thread.run(Thread.java:748) [?:?]

2019-06-08 16:55:11.942 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:6692a5df': null

java.lang.NullPointerException: null

	at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:103) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]

	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]

	at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [134:org.openhab.core:2.5.0.201906040301]

	at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [134:org.openhab.core:2.5.0.201906040301]

	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]

	at java.lang.Thread.run(Thread.java:748) [?:?]

I have same problem, on stable release 2.4.0 (openhabian on RPi2B) bluetooth binding works ok - BT dongle was detected and I could detect beacons. But I wanted to use RuuviTags binding so I switched to testing release 2.5.0M3. My BT dongle doesn't show up in Inbox. Messages similar as in thread above:
```
2019-10-16 11:55:54.321 [ERROR] [org.openhab.binding.bluetooth.bluez ] - bundle org.openhab.binding.bluetooth.bluez:2.5.0.M3 (268)[org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService(312)] : The activate method has thrown an exception
java.lang.NullPointerException: null

2019-10-16 11:55:54.530 [WARN ] [org.openhab.binding.bluetooth.bluez ] - FrameworkEvent WARNING - org.openhab.binding.bluetooth.bluez
org.osgi.framework.ServiceException: org.apache.felix.scr.impl.manager.SingleComponentManager.getService() returned a null service object

2019-10-16 11:55:54.537 [ERROR] [org.openhab.binding.bluetooth.bluez ] - bundle org.openhab.binding.bluetooth.bluez:2.5.0.M3 (268)[org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService(312)] : The activate method has thrown an exception
java.lang.NullPointerException: null

2019-10-16 11:55:54.920 [WARN ] [org.openhab.binding.bluetooth.bluez ] - FrameworkEvent WARNING - org.openhab.binding.bluetooth.bluez
org.osgi.framework.ServiceException: org.apache.felix.scr.impl.manager.SingleComponentManager.getService() returned a null service object

2019-10-16 11:55:54.927 [ERROR] [org.openhab.binding.bluetooth.bluez ] - bundle org.openhab.binding.bluetooth.bluez:2.5.0.M3 (268)[org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService(312)] : The activate method has thrown an exception
java.lang.NullPointerException: null

2019-10-16 11:55:55.308 [thome.event.ExtensionEvent] - Extension 'binding-bluetooth' has been installed.

2019-10-16 11:55:55.370 [hingStatusInfoChangedEvent] - 'bluetooth:bluez:hci0' changed from UNINITIALIZED (HANDLER_MISSING_ERROR) to INITIALIZING

2019-10-16 11:55:55.409 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler@100a96c': null
java.lang.NullPointerException: null

2019-10-16 11:55:55.421 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:hci0': null
java.lang.NullPointerException: null

2019-10-16 11:55:55.436 [hingStatusInfoChangedEvent] - 'bluetooth:bluez:hci0' changed from INITIALIZING to UNINITIALIZED (HANDLER_INITIALIZING_ERROR)

```

But in my case  results of `bundle:list | grep -i tinyb` is empty.
I tried  `bundle:install  https://janessa.me/esh/intel-iot-devkit.tinyb-0.5.1.jar`
but results is:
`
Error executing command: Error installing bundles:
        Unable to install bundle https://janessa.me/esh/intel-iot-devkit.tinyb-0.5.1.jar: org.osgi.framework.BundleException: Error occurred installing a bundle.
`
When I returned to 2.4.0 BT starts working again.
Switch to 2.5.0 SNAPSHOT results as for M3.

> 
> 
> @J-N-K do you what could explain the issues people are still seeing with 2.5.0M4?

Paper UI -> Add Bluetooth BlueZ Adapter -> Insert mac address

`2019-10-29 20:56:22.575 [hingStatusInfoChangedEvent] - 'bluetooth:bluez:e138f672' changed from UNINITIALIZED to INITIALIZING
2019-10-29 20:56:22.582 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler@1270f08': BlueZ JNI connection cannot be established.
java.lang.IllegalStateException: BlueZ JNI connection cannot be established.
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:98) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:152) [133:org.openhab.core:2.5.0.M4]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:52) [133:org.openhab.core:2.5.0.M4]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.lang.UnsatisfiedLinkError: tinyb.BluetoothManager.getNativeAPIVersion()Ljava/lang/String;
        at tinyb.BluetoothManager.getNativeAPIVersion(Native Method) ~[?:?]
        at tinyb.BluetoothManager.getBluetoothManager(BluetoothManager.java:304) ~[?:?]
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:93) ~[?:?]
        ... 10 more
2019-10-29 20:56:22.588 [hingStatusInfoChangedEvent] - 'bluetooth:bluez:e138f672' changed from INITIALIZING to UNINITIALIZED (HANDLER_INITIALIZING_ERROR): BlueZ JNI connection cannot be established.
2019-10-29 20:56:22.589 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:e138f672': BlueZ JNI connection cannot be established.
java.lang.IllegalStateException: BlueZ JNI connection cannot be established.
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:98) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:152) [133:org.openhab.core:2.5.0.M4]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:52) [133:org.openhab.core:2.5.0.M4]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: java.lang.UnsatisfiedLinkError: tinyb.BluetoothManager.getNativeAPIVersion()Ljava/lang/String;
        at tinyb.BluetoothManager.getNativeAPIVersion(Native Method) ~[?:?]
        at tinyb.BluetoothManager.getBluetoothManager(BluetoothManager.java:304) ~[?:?]
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:93) ~[?:?]
        ... 10 more
`

--- a/bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/package-info.java
+++ b/bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/package-info.java
@@ -10,7 +10,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-@org.osgi.annotation.bundle.Header(name = org.osgi.framework.Constants.BUNDLE_NATIVECODE, value = "lib/armv6hf/libjavatinyb.so;lib/armv6hf/libtinyb.so;processor=arm;osname=linux,*")
+@org.osgi.annotation.bundle.Header(name = org.osgi.framework.Constants.BUNDLE_NATIVECODE, value = "lib/armv6hf/libjavatinyb.so;lib/armv6hf/libtinyb.so;processor=arm;osname=linux,lib/x86-64/libjavatinyb.so;lib/x86-64/libtinyb.so;processor=amd64;osname=linux,*")
 package org.openhab.binding.bluetooth.bluez;
 
 /**

